doc/api/fs.md

```diff
@@ -1225,6 +1225,9 @@ changes:
 Change the file timestamps of a file referenced by the supplied file
 descriptor.
 
+*Note*: This function only works for AIX 7.1 and newer. It can still be
```

Comment:
And I think the second sentence makes it pretty clear that this is an AIX specific issue.